src/.gitignore

```diff
@@ -29,3 +29,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+###MAC Files###
+.DS_STORE
```

Comment:
@bbehling This needs to be ` .DS_Store`, entries are case-sensitive on Linux/Mac file systems.